brouter-core/src/main/java/btools/router/OsmPath.java

```diff
@@ -396,68 +396,66 @@ protected void addAddionalPenalty(OsmTrack refTrack, boolean detailMode, OsmPath
         traffic += dist*rc.expctxWay.getTrafficSourceDensity()*Math.pow(cost2/10000.f,rc.trafficSourceExponent);
       }
 
-      String wayKeyValues = "";
-      if ( message != null ) {
-        wayKeyValues = rc.expctxWay.getKeyValueDescription( isReverse, description );
-      }
+      // compute kinematic
+      computeKinematic( rc, dist, delta_h, detailMode );
 
-        if ( message != null )
-        {
-          message.turnangle = (float)angle;
-          message.time = (float)getTotalTime();
-          message.energy = (float)getTotalEnergy();
-          message.priorityclassifier = priorityclassifier;
-          message.classifiermask = classifiermask;
-          message.lon = lon2;
-          message.lat = lat2;
-          message.ele = ele2;
-          message.wayKeyValues = wayKeyValues;
-        }
+      if ( message != null )

```

Comment:
Github highlights a lot of diff here, but this is just fixing the indent which was one extra indent.